tests/PySys/mqtt_port/mqtt_port_unset/run.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import time
+
+from pysys.basetest import BaseTest
+
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system
+When mqtt.port is set using tedge mqtt.port set with sudo
+When the tedge config is listed and searched for the port that has been set 
+
+"""
+
+
+class MqttPortUnSet(BaseTest):
+    def setup(self):
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
```

Comment:
Could we not have a BaseTest where we have defined all our binaries variables, I see all of those being defined in each test case.

Reply:
I think this BaseTest is defined in the Pysys internally. I agree, we should put all these binaries in a single place and reuse them.

Reply:
Yes, you could do something similar as there is with `EnvironmentC8y`, create new class which inherits from `BaseTest`, but has base variables defined.

Reply:
Maybe a TedgeBaseTest class like I mentioned below? Could be new story together with adding a folder structure to group testcases.